homeassistant/components/homewizard/sensor.py

```diff
@@ -154,13 +169,29 @@ class HomeWizardSensorEntityDescription(SensorEntityDescription):
         enabled_fn=lambda data: data.total_energy_export_kwh != 0,
         value_fn=lambda data: data.total_energy_export_kwh,
     ),
+    HomeWizardSensorEntityDescription(
+        key="total_power_export_t1_kwh",
+        translation_key="total_energy_export_kwh",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        has_fn=lambda data: (  # HWE-SKT provides tarif 1 only
+            data.total_energy_export_t1_kwh is not None
+            and data.total_energy_export_kwh is None
+        ),
+        enabled_fn=lambda data: data.total_energy_export_t1_kwh != 0,
+        value_fn=lambda data: data.total_energy_export_t1_kwh,
+    ),
```

Comment:
```suggestion
```

Reply:
Fixed it during the rebase onto #104537 (as it caused merge conflicts/test conflicts)